cms/templates/partials/megamenu/navigation.html

```diff
@@ -1,9 +1,9 @@
-{% load menu_tags %}
+{% load menu_tags browse_tags %}
 
 {% for item in menu_items %}
   {% cycle '<ul class="main-menu-items">' '' '' '' %}
   <li>
-    <a href="">{{ item.text }}</a>
+    <a href="{{ item|url_for }}">{{ item.text }}</a>
```

Comment:
I wonder if it's worthwhile being more explicit with `{{ url 'browse' item.slug }}`? `url_for` in this context makes me expect the URL for the item itself, rather than browsing it.

Reply:
I've renamed it but we can't use the slug lookup because the item is a menu_item not a page. We need to use the link id to look up the item and build the slug.

I've renamed it